template/template.go

```diff
@@ -336,7 +356,11 @@ func (te Expander) Expand() (result string, resultErr error) {
 
 	templateTextExpansionTotal.Inc()
 
-	tmpl, err := text_template.New(te.name).Funcs(te.funcMap).Option("missingkey=zero").Parse(te.text)
```

Comment:
Given the usage inside Prometheus, can we just have Option as a string? I do not think it is worth the indirection to rebuild a string with = given that it will be static in prometheus.

Reply:
Perhaps we should do `[]string` as the function signature is `Option(opt ...string)`?

Reply:
yes sure